bats/1pk5col-ints.bats

```diff
@@ -398,32 +381,28 @@ if rows[2] != "9,8,7,6,5,4".split(","):
     dolt add test
     dolt commit -m "added test table"
     dolt branch test-branch
-    dolt table put-row test pk:0 c1:1 c2:2 c3:3 c4:4 c5:5
+    dolt sql -q "insert into test values (0, 1, 2, 3, 4, 5)"
     dolt add test
     dolt commit -m "added test row"
     dolt checkout test-branch
-    dolt table put-row test pk:0 c1:1 c2:2 c3:3 c4:4 c5:6
+    dolt sql -q "insert into test values (0, 1, 2, 3, 4, 6)"
     dolt add test
     dolt commit -m "added conflicting test row"
     dolt checkout master
     run dolt merge test-branch
     [ "$status" -eq 0 ]
     [[ "$output" =~ "CONFLICT (content)" ]]
-    run dolt table select test
+    run dolt sql -q "select * from test"
     [ "$status" -eq 0 ]
-    [[ "$output" =~ "Cnf" ]]
```

Comment:
We need a way to show conflicts exist when there are conflicts. I don't think we can delete these checks. Should select show conflicts for rows that have them? I think so